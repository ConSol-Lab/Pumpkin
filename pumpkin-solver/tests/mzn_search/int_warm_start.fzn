var 0..3: x:: output_var;
var 0..3: y:: output_var;
var 0..6: X_INTRODUCED_0_:: is_defined_var :: output_var;
array [1..2] of var int: variables = [x,y];
constraint int_lin_eq([1,1,-1],[x,y,X_INTRODUCED_0_],0):: defines_var(X_INTRODUCED_0_):: ctx_pos;
solve :: warm_start_int(variables,[1,2]) maximize X_INTRODUCED_0_;
