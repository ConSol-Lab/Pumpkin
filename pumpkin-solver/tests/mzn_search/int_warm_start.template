var 0..3: x;
var 0..3: y;

array[1..2] of var int: variables = [x, y];

solve :: warm_start(variables, [1, 2]) maximize x + y;
