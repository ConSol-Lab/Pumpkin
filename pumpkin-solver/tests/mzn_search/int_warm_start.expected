x = 1;
y = 2;
X_INTRODUCED_0_ = 3;
----------
x = 3;
y = 3;
X_INTRODUCED_0_ = 6;
----------
==========
